homepage.html

```diff
@@ -0,0 +1,73 @@
+                new. Pretty soon I found a passion for skateboarding. I was able to enjoy this activity by my rules.
+                Skateboarding taught me to never give up. We beat ourselves up for hours to achieve goals. Try, adjust, try again...
+                (sounds a lot like coding, right?). After a decade of skateboarding I packed up my belongings to move west into
+                the mecca of the American skate scene, Southern California. Living solo in California was a reality check
+                to say the least. It's an incredibly competitive economy and the cost of living is out of this world 
+                compared to North Carolina. I learned to cut costs by any means which involved: minimizing possessions, living in a van,
+                living on a boat, and taking any work opportunity. Four years had past and I had found myself a decent
+                job managing at a skate shop in historical Carlsbad, CA. Life was better than it had been, but I was
+                still working paycheck to paycheck and hardly skateboarding, the passion that had brought me to San Diego.
+                It was at this point that I knew I needed to make a major change. 
+            </p>
+            <h2>Why did I choose Software Development?</h2>
+            <p>The technical world isn't going away any time soon. All modern companies can be considered 
+                "tech companies", and without developers there is no "tech". These skills are in high 
+                demand. My decade-long involvement with the skateboard community led me to believe that software
+                development would be a great fit for me. What does skateboarding have to do with coding? Skateboarders
+                are relentless. We will give every ounce of energy, fall over and over to accomplish a goal that we
+                set for ourselves. It's trial and error, just like development. 
+                <br>
+                <br>
+                Towards the end of my recent stay in San Diego, I found Momentum. A fully immersive, three month program 
+                led by industry professionals; and it was ten minutes away from where I grew up! I plan to acquire the 
+                skills to understand both front and back end development in preparation for jobs in this industy. Excitement would 
+                be an understatement for how I feel about learning, building, and growing within the Durham technical community.
+            </p>
+
+        </article>
+
+        </div>
+    </body>
+    <footer>
+        <h2>Jacob Heivly
+            <br>
+            <em>919-360-4814</em>
+            <br>
+            <em>jfh708@gmail.com</em>
+        </h2>
+        <p>
+                <a href="https://www.facebook.com/jacob.heivly">
+                <img border="0" alt="W3Schools" src="https://image.ibb.co/izBREH/iconmonstr_facebook_4_48_1.png" width="50" height="50">
```

Comment:
Your `alt` attribute shows where you found this from. No problem looking around for solutions! Just review them and make sure you've edited them to do the right thing. These `alt` attributes should reflect the actual image you're showing.